entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Default to "unknown" if POD_NAME is not set
+pod_name=${POD_NAME:-unknown}
```

Comment:
Why don't you use the result of `hostname` here (which is the pod name on k8s machines)?

Reply:
Ah, true, thanks!